tla-io/src/test/scala/at/forsyte/apalache/io/quint/TestQuintEx.scala

```diff
@@ -220,4 +231,73 @@ class TestQuintEx extends AnyFunSuite {
     assert(exn.getMessage.contains(
             "Input was not a valid representation of the QuintIR: too many arguments passed to binary operator forall"))
   }
+
```

Comment:
I'm missing tests to `forall` and `exists`, specially one that tests them with multiple lambda arguments (I'm not sure if this would support that as is).

Reply:
Exists and forall were introduced in the previous PR, and you can see the teats for those on line 200/211 above.

I have totally overlooked the multi argument bindings of forall and exists, and I think of lambdas more generally! Thanks for pointing that out. I will add support for those in a follow up!